wazp/callbacks/metadata.py

```diff
@@ -373,6 +391,7 @@ def create_metadata_table_and_buttons(
                     generate_yaml_tooltip,
                 ]
             )
+        return None
```

Comment:
Not sure if we ever get into this situation? Should I throw an error? Or return an empty Div?